filebeat/docs/getting-started.asciidoc

```diff
@@ -99,6 +106,27 @@ To see the full list of variables for a module, see the documentation under
 
 include::{libbeat-dir}/shared/config-check.asciidoc[]
 
+[float]
+[[collect-application-logs]]
+==== Enable and configure ECS loggers for application log collection
+
+While {beatname_uc} can be used to ingest raw, plain-text application logs,
+we recommend structuring your logs at ingest time. This lets you extract fields,
+like log level and exception stack traces.
+
+Elastic simplifies this process by providing application log formatters in a variety
+of popular programming languages. These plugins format your logs into ECS-compatible JSON,
+which removes the need to manually parse logs.
+
+See {ecs-logging-ref}/intro.html[ECS loggers] to get started.
+
+[float]
+[[manual-configuration]]
+==== Configure {beatname_uc} manually
+
+If you're unable to find a module for your file type, or can't change your application's
```

Comment:
> or can't change your application's

Once we have implemented https://github.com/elastic/apm/issues/373, we might want to update that sentence and link to docs on how to use APM agents to automatically re-configure the app logging. This can be a solution for off-the-shelf apps like Atlassian products: While it's not possible or hard to add ecs-logging and re-configure the logging, the APM Agents can do that transparently.

Reply:
Makes sense – thanks for the heads-up.